cleanlab/datalab/issue_manager/label.py

```diff
@@ -63,6 +64,7 @@ def __init__(
         datalab: Datalab,
         clean_learning_kwargs: Optional[Dict[str, Any]] = None,
         health_summary_parameters: Optional[Dict[str, Any]] = None,
+        k: int = 10,
```

Comment:
Could you add this new argument to the docstring above explaining what it is for?

Reply:
Could you also add quick unittest to test passing `k` in as an arg in the `Datalab.find_issues` call?

Reply:
I am a bit confused by the expected behavior of passing `k` in as an arg in the Datalab.find_issues(). Should the expected behavior be an unexpected keyword argument? Since k is only defined in __init__().

Reply:
Thanks for the clarification! I was confused about how to pass `k`

Reply:
No worries! For more details you can also check out [this tutorial](https://docs.cleanlab.ai/stable/tutorials/datalab/datalab_advanced.html#Functionality-2:-Specifying-nondefault-arguments)